dcp_prototype/backend/wrangling/migrations/scripts/migrate_dcp_one_to_dcp_two.py

```diff
@@ -48,37 +46,61 @@ def gather_group_file_list(file_list):
     In particular, links need to be processed last.
     This is done so that all entities exist before linkage and linking will not
     occur between unknown entities.
+
+    All files should end with .json except link files
```

Comment:
Confused by this statement. Aren't links files named `links.json`?

Reply:
No Madison changed how all the files are named.
All entity files, except link files, look this this:
   UUID.DATE.json
All link files look like:
    UUID (but without dashes)

E.g.

this is an entity file (non-link)
eeb749e6-ce21-4bf4-a53a-005398a00862.2019-09-20T102839.972Z.json

this is a link file
f83094a5b135e7ca4ff9e2974ce36489

Reply:
Ahhh I see. Would you mind adding that as a comment; that was super helpful thanks!